src/main/java/org/folio/rest/impl/InventoryHelper.java

```diff
@@ -68,42 +68,49 @@
   static final String CONTRIBUTOR_NAME_TYPES = "contributorNameTypes";
   static final String INSTANCE_STATUSES = "instanceStatuses";
   static final String INSTANCE_TYPES = "instanceTypes";
-
   static final String ITEMS = "items";
+  static final String LOAN_TYPES = "loantypes";
```

Comment:
Following camelCase naming convention, should this be `loanTypes` ?

Reply:
This is in accordance with [inventory loantypes.json](https://github.com/folio-org/mod-inventory-storage/blob/master/ramls/loantypes.json#L6) so we cannot change the value

Reply:
Also, check API docs, its not camelCase https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/loan-type.html#loan_types_get